src/components/ReactGoogleMaps.js

```diff
@@ -268,6 +268,11 @@ export class ReactGoogleMaps extends Component {
                 className={"map"}
                 style={style}
                 zoom={this.state.zoom}
+                zoomControl={!isMobile}
+                streetViewControl={false}
+                mapTypeControl={false}
+                rotateControl={false}
+                fullscreenControl={false}
```

Comment:
There were some css rules that I removed that were hiding all of the map controls. Instead these props will hide them. The zoom control will only be visible for desktop. Also, there is some Google branding that was being hidden by the css rules that is now visible at the bottom of the page. I believe that technically we are required to display these things as part of the terms of service.